### PR TITLE
Create pagination button

### PR DESCRIPTION
**Pull request by BenjaminParment**
-> Adds pagination button
-> Same sort of form as other components, should I add a quick test?

### CODE REVIEW

**Comment by BenjaminParment** [comment]
I added the functionality to not trigger the callback function when the page is already selected and set the default cursor for those

File: src/components/pagination-button/PaginationButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            jestFuntion = jest.fn();
+            wrapper = shallow(<PaginationButton onClick={jestFuntion} selected={false} value={pageValue} />);
+        });
+
+        describe("when the button is clicked", () => {
+            beforeEach(() => {
+                wrapper.find(".pagination-button").simulate("click");
+            });
+
+            it("calls the finctions passed as props", () => {
```
Comment (j-n-white) [inline src/components/pagination-button/PaginationButton.test.tsx]: Try and stick to Given, When, Then descriptions
```suggestion
            it("then it calls the function passed as props", () => {

```

File: src/components/pagination-button/PaginationButton.test.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+describe("Given a pagination button", () => {
+    const pageValue = 1;
+    let jestFuntion: jest.Mock;
+    let wrapper: ShallowWrapper;
+
+    describe("that is not selected", () => {
+        beforeEach(() => {
+            jestFuntion = jest.fn();
+            wrapper = shallow(<PaginationButton onClick={jestFuntion} selected={false} value={pageValue} />);
```
Comment (j-n-white) [inline src/components/pagination-button/PaginationButton.test.tsx]: You could create the shallow render in a before each at the root describe and then here just set the part of the props you are interested in.

File: src/components/pagination-button/PaginationButton.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    private paginationClick = () => {
+        if (!this.props.selected) {
+            this.props.onClick(this.props.value);
+        }
+    };
+
+    render = () => (
+        <button
+            className={`pagination-button ${this.props.selected ? "pagination-button-selected" : ""}`}
+            onClick={this.paginationClick}
```
Comment (j-n-white) [inline src/components/pagination-button/PaginationButton.tsx]: aria text

File: src/sandbox/Sandbox.tsx [hunk 4]
```diff
@@ -117,6 +122,10 @@ class Sandbox extends React.Component<ConnectedSandboxProps, SandboxState> {
                 />
                 <TextButton value={"Text2"} onClick={this.onClickTextButton} />
             </div>
+            <div className="pagination">
+                <PaginationButton value={1} onClick={this.onClickPagination} selected={this.state.text === "1"} />
```
Comment (j-n-white) [inline src/sandbox/Sandbox.tsx]: This selected logic is basically going to be the same every time so how about passing through a selected page on the props?

**Pull request merged**